Giveaways.md

```diff
@@ -0,0 +1,15 @@
+# Giveaways
+From time to time Devstaff has the opportunity to perform some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+
+Here we describe the process, the tools and the rules we use for these giveaways.
+
+# Prizes list
+All prizes will be written down at each meetup's announcement at [meetup.com](https://www.meetup.com/devstaff/). Moreover, at the intro presentation, when sponsors are presented, each prize will be explicitly mentioned as far as the type, the amount of prizes and the sponsor of each one.
+
+# Participants eligibility
+All participants of a meetup are eligible for the raffling process. Org team member's that atend the meetup (we prefer to provide more prizes to the community and have a chance to win them rather than benefit things as org teams members 😉), speaker(s) as well as meetup's attendees will participate in the raffling process. The participants of a meetup are the members that have joined our meetup's place (in case of physical meetups) or our communication tool (in case of remote meetups).
```

Comment:
Can you please also make `attend` instead of atend

Reply:
Just updated by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/462/commits/b5477d5ab73b8780e5140cb690798434d8fcf254